src/test/java/org/jenkinsci/plugins/workflow/test/steps/FakeStep.java

```diff
@@ -0,0 +1,69 @@
+package org.jenkinsci.plugins.workflow.test.steps;
+
+import java.io.Serializable;
+import java.util.Set;
+
+import org.jenkinsci.plugins.workflow.steps.Step;
+import org.jenkinsci.plugins.workflow.steps.StepContext;
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor;
+import org.jenkinsci.plugins.workflow.steps.StepExecution;
+import org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import hudson.Extension;
+import hudson.model.TaskListener;
+
+public class FakeStep extends Step implements Serializable {
```

Comment:
Just pass the `int sleepSeconds` to the `FakeExecution` constructor.

Better yet, use https://javadoc.jenkins.io/plugin/workflow-step-api/org/jenkinsci/plugins/workflow/steps/StepExecutions.html#synchronousNonBlocking(org.jenkinsci.plugins.workflow.steps.StepContext,org.jenkinsci.plugins.workflow.steps.StepExecutions.SynchronousBody)

Reply:
thank you, nice. fixed using lambda